packages/replay/src/integration.ts

```diff
@@ -185,6 +185,17 @@ Sentry.init({ replaysOnErrorSampleRate: ${errorSampleRate} })`,
     this._replay.stop();
   }
 
+  /**
+   * Immediately send all pending events.
+   */
+  public flush(): Promise<void> | void {
+    if (!this._replay || !this._replay.isEnabled()) {
+      return;
+    }
```

Comment:
l: Just wondering if we should check for `isPaused()` here as well... technically, recording could be paused b/c of rate limiting and we wouldn't want to flush in this case. But overall, I think chances are pretty low that this would actually happen. And even if it does, the consequences wouldn't be bad, as we're already handling rate limit responses while being rate-limited. WDYT?

Reply:
Yeah, I thought the same. I'd say it is OK, as (theoretically) when paused the event buffer has to be empty always, so it wouldn't even attempt to send anything. I'd say it is OK to leave it like this?

Reply:
> when paused the event buffer has to be empty always

Not necessarily, pause just stops DOM recording, we could have events from `window.performance`, but in that case, might be ok to try to flush.

Reply:
hmm yeah, true, I'd say that is not incorrect to flush the stuff that has not been sent yet due to being paused?